modules/storage-expanding-recovering-failure.adoc

```diff
@@ -7,13 +7,4 @@
 [id="expanding-recovering-from-failure_{context}"]
 = Recovering from failure when expanding volumes
 
-If expanding underlying storage fails, the {product-title} administrator can manually recover the persistent volume claim (PVC) state and cancel the resize requests. Otherwise, the resize requests are continuously retried by the controller.
-
-.Procedure
-
-. Mark the persistent volume (PV) that is bound to the PVC with the `Retain` reclaim policy. This can be done by editing the PV and changing `persistentVolumeReclaimPolicy` to `Retain`.
-. Delete the PVC.
-. Manually edit the PV and delete the `claimRef` entry from the PV specs to ensure that the newly created PVC can bind to the PV marked `Retain`. This marks the PV as `Available`.
-. Re-create the PVC in a smaller size, or a size that can be allocated by the underlying storage provider.
-. Set the `volumeName` field of the PVC to the name of the PV. This binds the PVC to the provisioned PV only.
-. Restore the reclaim policy on the PV.
+If a resize request fails or or remains in a pending state, you can try again by entering another smaller resize value in `.spec.resources.requests.storage` for the persistent volume claim (PVC).
```

Comment:
Sounds obvious but the "smaller value" should not be smaller than the original volume size.

Reply:
`fails or or remains ` there are two `or`

Reply:
I think that manual process should still be documented as an alternative btw.

Reply:
Done.